quarkus-service-registry/README.adoc

```diff
@@ -0,0 +1,249 @@
+endif::[]
+
+[id="proc-importing-quarkus-registry-sample-code_{context}"]
+== Importing the Quarkus sample code
+
+[role="_abstract"]
+For this quick start, you'll use the Quarkus {registry} sample code from the App Services {samples-git-repo}[Guides and Samples^] repository in GitHub. After you understand the concepts and tasks in this quick start, you can use your own Quarkus applications with {product} and {registry} in the same way.
+
+.Procedure
+. On the command line, clone the App Services {samples-git-repo}[Guides and Samples^] repository from GitHub.
++
+.Cloning the guides and samples repository
+[source,subs="+attributes"]
+----
+git clone {samples-git-repo} app-services-guides
+----
+. In your IDE, open the `code-examples/quarkus-service-registry-quickstart` directory from the repository that you cloned.
+
+ifdef::qs[]
+.Verification
+* Is the Quarkus example application accessible in your IDE?
+endif::[]
+
+[id="proc-configuring-quarkus-registry-app_{context}"]
+== Configuring the Quarkus application to connect to Kafka and {registry} instances
+
+[role="_abstract"]
+To enable your Quarkus application to access a Kafka instance, configure the connection properties using the Kafka bootstrap server endpoint. To access a {registry} instance, configure the registry endpoint connection property with the Core Registry API value. 
+
+Access to the {registry} and Kafka instances is managed using the same service account and SASL/OAUTHBEARER token endpoint. For Quarkus, you can configure all connection properties using the `application.properties` file. This example sets environment variables and references them in this file.
+
+Quarkus applications use https://github.com/eclipse/microprofile-reactive-messaging[MicroProfile Reactive Messaging^] to produce messages to and consume messages from your Kafka instances in {product}. For details on configuration options, see https://quarkus.io/guides/kafka[Using Apache Kafka with Reactive Messaging^] in the Quarkus documentation.
+
+This Quarkus example includes both consumer and producer applications.
+
+.Prerequisites
+ifndef::qs[]
+* You have the Kafka bootstrap server endpoint for the Kafka instance. You copied this information previously for the Kafka instance in {product} by selecting the options menu (three vertical dots) and clicking *Connection*.
+* You have the Core Registry API endpoint for the {registry} instance. You copied this information for the {registry} instance  by selecting the options menu (three vertical dots) and clicking *Connection*. From the list of endpoints, you copied the *Core Registry API* that is supported by the Apicurio serializer/deserializer (SerDes) used in this example. 
+* You copied the *Token endpoint URL* from the same list of endpoints to be used for the OAuth based athentication method used in this example.
+* You have generated a service account from the Service Accounts section of {product} and stored in a secure place.
```

Comment:
I said nothing, the service account is present here.

Reply:
Yes, good point, bumping this up...